lib/media/streaming_engine.js

```diff
@@ -1240,7 +1240,15 @@ shaka.media.StreamingEngine = class {
       if (this.config_.lowLatencyMode && isReadableStreamSupported && isMP4 &&
           !reference.hlsAes128Key) {
         let remaining = new Uint8Array(0);
+        let processingResult = false;
+        let callbackCalled = false;
         const streamDataCallback = async (data) => {
+          if (processingResult) {
+            // If the fallback result processing was triggered, don't also
```

Comment:
Am I correct in my understanding, that this shouldn't happen?  In theory, if the fallback has executed, the streamDataCallback shouldn't fire _after_ that, correct?

Reply:
The theory tells me that it should never happen, but... I have found a case in an old TV :(

Reply:
Oh, yikes.  Well, maybe add some mention of that in the comments.  Otherwise, some well-meaning person might "optimize" this check away later.

Reply:
Added!